Template/deviceConfiguration.json

```diff
@@ -31,9 +31,6 @@
               "mqttSettings__enabled": {
                 "value": "false"
               },
-              "httpSettings__enabled": {
```

Comment:
Did you had to remove this because of the logging?

Reply:
This was because the metrics collector scrapes metrics on `http://edgeHub:9600/metrics` on the hub. When we disable HTTP we can't do that.